django_react_proj/users-fe/src/components/NewUserForm/NewUserForm.js

```diff
@@ -47,15 +68,33 @@ const NewUserForm = ({ user, resetState, toggle }) => {
         <Input type="text" name="name" onChange={onNameChange} value={defaultIfEmpty(name)} />
       </FormGroup>
       <FormGroup>
-        <Label for="deadline">Data wykonania:</Label>
+        <Row form>
+          <Col>
+            <Label for="deadline">Data wykonania:</Label>
+          </Col>
+          <Col>
+            <Label for="deadline">Task type</Label>
+          </Col>
+        </Row>
         <FormGroup>
-          <TimePicker
-            format="H:m"
-            disableClock="true"
-            name="deadline"
-            onChange={setDeadline}
-            value={defaultIfEmpty(deadline)}
-          />
+          <Row>
+            <Col>
+              <TimePicker
+                format="H:m"
+                disableClock="true"
+                name="deadline"
+                onChange={setDeadline}
+                value={defaultIfEmpty(deadline)}
+              />
+            </Col>
+            <Col sm={7}>
+              <Input type="select" onChange={onTaskTypeChange} value={defaultIfEmpty(taskType)}>
+                {types.map((type) => (
+                  <option name="taskType">{type.type}</option>
```

Comment:
components rendered inside `map` should have key attribute which must be unique.